libs/utils/android/screen.py

```diff
@@ -136,4 +137,20 @@ def get_screen_density(target):
         """
         return target.execute('getprop ro.sf.lcd_density')
 
+    @staticmethod
+    def set_screen(target, on=True):
+        log = logging.getLogger('Screen')
+        if on == True:
+            target.execute('input keyevent KEYCODE_WAKE')
+            log.info('Setting screen ON')
+        else:
```

Comment:
I would prefer:

```python
if not on:
    System.sleep()
    log.info('Setting screen OFF')
    return

System.wake()
log.info('Setting screen ON')
```

Which better match the API signature where also the non default case returns immediately.